src/org/mangui/hls/stream/StreamBuffer.as

```diff
@@ -350,14 +351,15 @@ package org.mangui.hls.stream {
                         break;
                     case FLVTag.METADATA:
                         _metaTags.push(tagData);
-                        if (tag.captionData) {
```

Comment:
We should probably remove the old captionData element that was added in https://github.com/gesinger/flashls/commit/f0befd7e15ff134638cf8e04666a3bc3fbbbc707